docs/modules/cluster-performance/pages/best-practices.adoc

```diff
@@ -1507,6 +1507,7 @@ hazelcast:
       backup-count: 1
 ----
 ====
+--
```

Comment:
Why is this necessary? I haven't seen it mentioned on https://github.com/hazelcast/hz-docs/blob/main/docs-tabs-library/tabs-block.js